### ISSUE TITLE
Hang on startup if map_config.conf is invalid

### ISSUE THREAD

**Issue by DerDakon** [M4rtinK/modrana#138]
I had a stray " in the label field of a map_config.conf entry, which resulted in the application only showing the progress bar on startup and never anything else. This was on my Jolla phone.

**Comment by M4rtinK** [comment]
Yep, that's certainly wrong and should be fixed - I'm thinking about hardcoding at least Mapnik and using it as a fallback if the map config file fails to parse or is unavailable for some reason. 

Also I guess I should also check if ConfigObj can do partial parsing & ignore individual syntax errors.

But eventually I would like to get the map layer management more robust:
- add support for a map layer config drop directory, eq. `map_config.d/` where users could easily drop their additional map layer snippets that modRana would add on top of `map_config.conf`
- support for interactive map layer editing in the UI - this would be probably implemented by adding a config snippet into the drop directory mentioned above if the user edits or adds a layer in the UI

So that's the plan, I just have not really got to that yet myself. :)
